### PR TITLE
Add ua-parser 1.3.0.wso2v2

### PR DESCRIPTION
**Pull request by gayashanbc**
Related https://github.com/wso2/carbon-identity-framework/pull/2404

### CODE REVIEW

File: ua-parser/1.3.0.wso2v2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!--
+ ~ Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline ua-parser/1.3.0.wso2v2/pom.xml]: Shouldn't year change?

File: ua-parser/1.3.0.wso2v2/pom.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    <artifactId>ua-parser</artifactId>
+    <packaging>bundle</packaging>
+    <name>ua-parser-wso2</name>
+    <version>1.3.0.wso2v2</version>
+    <description>
+        This bundle represents ua-parser 1.3.0
+    </description>
+    <url>http://wso2.org</url>
+
+    <repositories>
```
Comment (thusithathilina) [inline ua-parser/1.3.0.wso2v2/pom.xml]: Does this really need?

File: ua-parser/1.3.0.wso2v2/pom.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            <name>Twitter Public Maven Repo</name>
+            <url>http://maven.twttr.com</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>ua_parser</groupId>
+            <artifactId>ua-parser</artifactId>
+            <version>1.3.0</version>
```
Comment (thusithathilina) [inline ua-parser/1.3.0.wso2v2/pom.xml]: Shouldn't this be optional?

File: ua-parser/1.3.0.wso2v2/pom.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    </build>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline ua-parser/1.3.0.wso2v2/pom.xml]: No need of snapshot distro

File: ua-parser/1.3.0.wso2v2/pom.xml [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Bundle-Vendor>WSO2 Inc</Bundle-Vendor>
+                        <Bundle-Description>${project.description}</Bundle-Description>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Export-Package>
+                            ua_parser.*;version="${project.version}",
+                            org.yaml.snakeyaml.*;version="${project.version}"
```
Comment (thusithathilina) [inline ua-parser/1.3.0.wso2v2/pom.xml]: Why does this export snakeyaml?

File: ua-parser/1.3.0.wso2v2/pom.xml [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+                        </Export-Package>
+                        <Import-Package>
+                            org.apache.commons.collections.map;version="[3.2.0,3.3.0)"
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (thusithathilina) [inline ua-parser/1.3.0.wso2v2/pom.xml]: Is this really needed?

File: ua-parser/1.3.0.wso2v2/pom.xml (hunk 1 above)
Comment (gayashanbc) [reply ua-parser/1.3.0.wso2v2/pom.xml]: Fixed in https://github.com/wso2/orbit/pull/393

File: ua-parser/1.3.0.wso2v2/pom.xml (hunk 3 above)
Comment (gayashanbc) [reply ua-parser/1.3.0.wso2v2/pom.xml]: Fixed in https://github.com/wso2/orbit/pull/393

File: ua-parser/1.3.0.wso2v2/pom.xml (hunk 4 above)
Comment (gayashanbc) [reply ua-parser/1.3.0.wso2v2/pom.xml]: Removed the snapshot distro in https://github.com/wso2/orbit/pull/393

File: ua-parser/1.3.0.wso2v2/pom.xml (hunk 6 above)
Comment (gayashanbc) [reply ua-parser/1.3.0.wso2v2/pom.xml]: Removed in https://github.com/wso2/orbit/pull/393

File: ua-parser/1.3.0.wso2v2/pom.xml (hunk 5 above)
Comment (gayashanbc) [reply ua-parser/1.3.0.wso2v2/pom.xml]: `snakeyaml` is a dependancy to the `Parser` class that resides in the `ua_parser` package.
Hence, this is required.

File: ua-parser/1.3.0.wso2v2/pom.xml (hunk 2 above)
Comment (gayashanbc) [reply ua-parser/1.3.0.wso2v2/pom.xml]: Yes, this is required.

**Pull request merged**